query/aql_processor.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	hllQueryRequiredMemoryInMB = 10 * 1024
+	hllQueryRequiredMemoryInBytes = 10 * (1 << 30)
```

Comment:
so originally we only reserved 10K for HLL? and now 10GB, isn't a little too bigger?

Reply:
yeah we have been reserved 10K.
10GB is to make sure each card only run one hll at a time since we cannot estimate hll effectively

Reply:
sure, we better do 2 observation:
1. how many real gpu memory allocated when running weekly hll contractor in production/staging
2. how it impacts production query when hll runs with 10GB reserved. Now as we under-reserve the memory, so other queries may still able to run. After we bump to 10G, other queries in the same machine will be totally blocked even the GPU might have spare space.